.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+__*This is the standard PR description template.*__
+
+__*Delete the parts of this template that aren’t relevant to the current PR you are submitting – for example, you may not need to amend documentation or testing of the component.*__
+
+---
+
+_A short description of your PR goes here._
+
+## UI Review Checks
+
+![image](https://media.github.je-labs.com/user/326/files/c1e6632c-e23e-11e6-91e2-4625e4f89d6d)
+
+
+- [ ] This PR has been checked with regard to our brand guidelines
+- [ ] UI Documentation has been [created|updated]
+- [ ] This code has been checked with regard to our accessibility standards
+  - [ ] HTML is valid [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.8hhpeziqc7vf)
+  - [ ] HTML is well structured [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.mwt4cyjxbtli)
+  - [ ] HTML is well ordered [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.q114rm3u5ix5)
+  - [ ] HTML is semantic [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.9r5q60p40cca)
+  - [ ] HTML is labelled [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.4n7zr152hxh7)
+  - [ ] Passed an accessibility audit [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.9yn7a4j48ztn)
+  - [ ] Keyboard is supported [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.is5np1jwp1mp)
+  - [ ] Tab stops in place [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.xbwmm6t7axsn)
+  - [ ] Focus is managed [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.14zt9gvd027b)
+  - [ ] States are updated [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.iv5nfqs2k0n2)
+  - [ ] Page is well spoken [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.lrcxo6brcnvx)
+
+## Browsers Tested
+
+- [ ] Chrome
+- [ ] Edge
+- [ ] Internet Explorer 11
+- [ ] Mobile (i.e. iPhone/Android - please list device)
+
+### List any other browsers that this PR has been tested in:
+
+- [View the Browser Support Checklist](https://docs.google.com/spreadsheets/d/1GqlRuEgtqqmMiIiPBCE39nLq8hWdzayRoYFzvTt17YQ/edit#gid=0)
```

Comment:
Is this doc publicly readable?

Reply:
No, it isn't

Reply:
I'll add this into the docs

Reply:
[Added in this PR](https://github.com/justeat/global-component-library/pull/30)